### PR TITLE
feat(ui): Add unhandled label to issue stream + detail

### PR DESCRIPTION
**Pull request by matejminar**
This feature is behind the feature flag for internal testing.

image

image

Depends on https://github.com/getsentry/sentry/pull/20519

### CODE REVIEW

**Comment by matejminar** [comment]
There will be a slight change where we won't use the new Label component, but rather the Tag component.
https://github.com/getsentry/sentry/pull/20936

**Review by markstory — APPROVED** [review]
Looks good for internal testing.

File: src/sentry/static/sentry/app/views/organizationGroupDetails/unhandledTag.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  margin-right: ${space(1)};
+`;
+
+const TagAndMessageWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+`;
+
+// TODO(matej): remove "unhandled-issue-flag" feature flag once testing is over (otherwise this won't ever be rendered in a shared event)
+const UnhandledTag = styled(props => (
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/unhandledTag.tsx:21]: should we add a type to this `props`?

**Review by priscilawebdev — APPROVED** [review]
It looks good! :)

**Pull request merged**